openstates/scrape/base.py

```diff
@@ -196,6 +205,56 @@ def push_to_queue(self):
         )
         self.info(f"Message ID: {response['MessageId']}")
 
+    def _force_archive_jsonl_to_gcs(self):
+        cloud_storage_client = storage.Client(project=GCP_PROJECT)
+        bucket = cloud_storage_client.bucket(BUCKET_NAME)
+
+        for data_class in self._data_classes:
+            jsonl_path = os.path.join(self.datadir, f"{data_class}.jsonl")
+            if os.path.exists(jsonl_path) and os.path.getsize(jsonl_path) > 0:
+                timestamp = datetime.datetime.now(datetime.timezone.utc).isoformat()
+                jurisdiction_id = self.jurisdiction.jurisdiction_id.replace(
+                    "ocd-jurisdiction/", ""
+                )
+                dest_file_path = f"{SCRAPE_LAKE_PREFIX}/realtime/{jurisdiction_id}/{data_class}_{timestamp}.jsonl"
```

Comment:
Two things here:

1. I don't know if it throws a wrench in processing downstream, but if not it might be worth adding "YYYY-MM` to the object path, just to make manual browsing easier (we've already felt this pain in the existing archive path naming in GCS). So something like `{SCRAPE_LAKE_PREFIX}/realtime/{jurisdiction_id}/{date_format(now, 'YYYY-MM')}/{data_class}_{timestamp}.jsonl` (obv that's pseudo code)
2. I think "realtime" should be part of `SCRAPE_LAKE_PREFIX` so that it fully comes from environment config, not hardcoded here